lightning/src/ln/chan_utils.rs

```diff
@@ -551,36 +563,47 @@ pub fn build_htlc_transaction(prev_hash: &Txid, feerate_per_kw: u32, contest_del
 pub struct ChannelTransactionParameters {
```

Comment:
This particular struct is definitely *both* parties' parameters.  The counterparty ones are specifically broken out into a separate struct because they are populated later.